### ISSUE TITLE
Question:  Can formvalidation work with a link (<a href="url" ... />) instead of button

### ISSUE THREAD

**Issue by boulepick** [formvalidation/support#835]
Can you tell me if it is possible to get formvalidation to work with a link instead of a button.

**Comment by CSADn** [comment]
What you mean?
Do you want that an anchor fire the validation of all the fields in a form?

**Comment by boulepick** [comment]
yes instead of a the submit button to fire the validation, it would be the anchor to fire the validation. that way we can define specific validations for an anchor or a type of anchor.
An example would be having a form with a register with Facebook link and a local register through a form. for the register with facebook link certain fields needs to be validated before the user can go through and for the local registration all the fields needs to be validated.

**Comment by CSADn** [comment]
Actually it's possible because formValidation starts validate all fields when onSubmit is fired. So you can actually fire that event on any html element through the javascript events like click. For example if you want an anchor to fire submit, just bind click event to it and call submit of the form in his callback. If you are using jQuery it's like:

```javascript
   $('a').on('click', function() {
      $('form').submit();
   });
```
Hope this help you.
